File: API/Models/Entities/QuestionSet.cs
namespace API.Models.Entities {
    public class QuestionSet {
        public int QuestionID { get; set; }
        public string Source { get; set; }
        public string Key { get; set; }
        public string Type { get; set; }
        public string TypeDetail { get; set; }
        public string Question { get; set; }
        public string Options { get; set; }
        public int? MinValue { get; set; }
        public int? MaxValue { get; set; }
    }
}

File: client/src/app/models/form-models/questionSet.ts
// export interface QuestionSet {
//     questionId: number;
//     source: string;
//     key: string;
//     type: string;
//     typeDetail: string;
//     question: string;
//     options: [];
//     minValue: number;
//     maxValue: number;
// }

export interface QuestionSet {
    tableName: string;
    key: string;
    label: string;
    questionOrder: number;
    type: string;
    typeDetail: string;
    minValue: number;
    maxValue: number;
}